runestone/common/project_template/pavement.tmpl

```diff
@@ -50,6 +50,7 @@ options(
             # These are used for non-dynamic books.
             'appname': 'runestone',
             'course_url': master_url,
+            'language': 'en', 
```

Comment:
Where is this used?

Reply:
gotcha -- just wanted to make sure it was not conflicting with any of our other uses of language